docs/POLYFILLS.md

```diff
@@ -0,0 +1,117 @@
+# Using Polyfills
+
+In the case you would like to use Internet Explorer, you might be interested in polyfills. 
```

Comment:
Let's be more politic about this :p  "In the case you want your application to be compatible with non modern browsers (i.e. Internet Explorer) or browsers that don't support some of the latest standards, ..."